README.md

```diff
@@ -563,13 +563,12 @@ We recommend the following software to use when getting to know GS1 Resolver and
 
 8. Now wait 10 seconds while the system settles down (the SQL Server service takes a few seconds to initialise when '
    new') then, if you have SQL Server Management Studio installed, go to Step 9, else copy and paste this command which
-   will cause you to enter the container and access its terminal prompt:<pre>docker exec -it resolver-sql-server bash</pre>
-   Now run this command which will create the database and some example data:<pre>/opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P its@SECR3T! -i /gs1resolver_sql_scripts/sqldb_create_script.sql </pre>
+   will copy the initialization SQL script into the database container:<pre>docker compose cp ./resolver_sql_server/sqldb_create_script.sql sql-server:sqldb_create_script.sql</pre>
```

Comment:
this didn;t work for me but this did.

docker cp ./resolver_sql_server/sqldb_create_script.sql resolver-sql-server:sqldb_create_script.sql

Reply:
@paulfdietrich can you show me the exact error message? I think you should install docker compose

Reply:
seems like [`docker compose cp`](https://docs.docker.com/engine/reference/commandline/compose_cp/) is only supported in compose v2(the one without `-`) or highter: https://www.docker.com/blog/announcing-compose-v2-general-availability/